flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -19,7 +19,10 @@
 package org.apache.flink.table.api.scala.stream.sql
 
 import org.apache.flink.api.scala._
-import org.apache.flink.table.api.scala.stream.sql.SqlITCase.EventTimeSourceFunction
+import org.apache.flink.table.api.scala.stream.sql.SqlITCase.{
+  EventTimeSourceFunction, 
+  RowResultSortComparator,
+  TupleRowSelector}
```

Comment:
Done...but i must admit i am confused with the application of the style. I thought we go for lines of less than 100 characters...why would this be an exception?

@sunjincheng121